tests/test_result.py

```diff
@@ -24,10 +25,12 @@ def test_json_error_serialization() -> None:
         error = exc
 
     task = TaskiqResult(is_err=False, return_value=1, execution_time=0, error=error)
-    data: TaskiqResult[int] = TaskiqResult.parse_raw(task.json())
```

Comment:
made this change since `pydantic v2` is stricter, we can't unserialize back a custom serializer